src/test/java/com/amazon/ion/impl/IonRawTextReaderTest_1_1.java

```diff
@@ -126,4 +128,29 @@ public void invalidExpressionSyntax(int minorVersion, String input, String first
             assertThrows(IonReaderTextRawX.IonReaderTextParsingException.class, reader::nextRaw);
         }
     }
+
+    @ParameterizedTest
+    @CsvSource({
+        "(:values 0) (:values 1)",
```

Comment:
If I recall correctly, this is only a reference to a macro in the local macro table, and the system macros are loaded into the local macro table when the Ion 1.1 IVM is encountered, so this test case is correct.

However, I think I need to understand the intention of this test. I was expecting to see a test case for the System E-Expression syntax, rather than invoking a system macro via the local macro table. Which was your intention?